scvi/data/_built_in_data/_cellxgene.py

```diff
@@ -0,0 +1,82 @@
+import os
+from typing import Optional
+
+import pandas as pd
+import requests
+from anndata import AnnData, read_h5ad
+
+from scvi.data._download import _download
+
+CELLXGENE_PRODUCTION_HOST = "api.cellxgene.cziscience.com"
+CELLXGENE_PRODUCTION_ENDPOINT = f"https://{CELLXGENE_PRODUCTION_HOST}"
+DATASETS = f"{CELLXGENE_PRODUCTION_ENDPOINT}/dp/v1/datasets/"
+COLLECTIONS = f"{CELLXGENE_PRODUCTION_ENDPOINT}/dp/v1/collections/"
+CELLXGENE_EXPLORER = "https://cellxgene.cziscience.com/e/"
+
+
+def _presign_url(url: str):
+    resp = requests.post(url)
+    return resp.json()["presigned_url"]
+
+
+def _load_cellxgene_dataset(
```

Comment:
should we link isaac's gist where this code is inspired from?

Reply:
I don't use anything from it anymore